internal/github.com/hyperledger/fabric/bccsp/sw/vaultkeystore.go

```diff
@@ -0,0 +1,343 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
```

Comment:
Who developed significant aspects of this file.

Reply:
Yes, there is copyright here.

Reply:
The Copyright statement should be of the contributor.